fenix/app/src/main/java/org/mozilla/fenix/browser/BaseBrowserFragment.kt

```diff
@@ -210,7 +209,9 @@ abstract class BaseBrowserFragment :
     internal val browserToolbarView: BrowserToolbarView
         get() = _browserToolbarView!!
 
-    internal lateinit var browserToolbar: BrowserToolbar
+    private var _bottomToolbarContainerView: BottomToolbarContainerView? = null
```

Comment:
❓ Not for this PR but maybe as a follow up we rename `BottomToolbarContainerView`?  Looks like it's more a `NavbarContainerView`?